File: Api/Timesheet/Entities/Accesscontrol/Asset.cs
namespace Timesheet.Entities.Accesscontrol
{
    public class Asset : EntityBase
    {
        public string Code { get; set; }
        public string Name { get; set; }
        public string DisplayName { get; set; }
        public int? AssetTypeId { get; set; }
        public AssetType? AssetType { get; set; }
        public bool IsGlobal { get; set; }
        public string GroupName { get; set; }
        public string IconClass { get; set; }
        public string RouterLink { get; set; }
        public string TranslationKey { get; set; }

    }
}

File: Web/src/app/models/access-control/asset.ts
import { LegalEntityLite } from '../legal-entity-lite';
import { AssetType } from './asset-type';

export class Asset {
  AssetType: AssetType;
  AssetTypeId: number;
  Code: string;
  DisplayName: string;
  Id: number;
  IsGlobal: boolean;
  LegalEntity: LegalEntityLite;
  Name: string;
  GroupName: string;
  IconClass: string;
  RouterLink: string;
  TranslationKey: string;

  constructor(ob: any) {
    this.AssetType = ob.AssetType;
    this.Code = ob.Code;
    this.DisplayName = ob.DisplayName;
    this.Id = ob.Id;
    this.IsGlobal = ob.IsGlobal;
    this.LegalEntity = ob.LegalEntity;
    this.Name = ob.Name;
    this.GroupName = ob.GroupName;
    this.AssetTypeId = ob.AssetTypeId;
    this.IconClass = ob.IconClass;
    this.RouterLink = ob.RouterLink;
    this.TranslationKey = ob.TranslationsKey;
  }
}
